Doppler.BillingUser/Infrastructure/BillingRepository.cs

```diff
@@ -586,6 +586,30 @@ LEFT JOIN
             return result;
         }
 
+        public async Task UpdateBillingCreditAsync(int billingCreditId, BillingCreditPaymentInfo billingCreditPaymentInfo)
+        {
+            using var connection = _connectionFactory.GetConnection();
+            await connection.QueryFirstOrDefaultAsync(@"
+UPDATE
+    BC.CCNumber = @CCNumber,
+    BC.CCExpMonth = @CCExpMonth,
+    BC.CCExpYear = @CCExpYear,
+    BC.CCVerification = @CCVerification
+FROM
+    [dbo].[BillingCredits] BC
+WHERE
+    IdBillingCredit = @billingCreditId",
+                new
+                {
+                    @billingCreditId = billingCreditId,
+                    @CCNumber = _encryptionService.EncryptAES256(billingCreditPaymentInfo.CCNumber.Replace(" ", "")),
+                    @CCExpMonth = billingCreditPaymentInfo.CCExpMonth,
+                    @CCExpYear = billingCreditPaymentInfo.CCExpYear,
+                    @CCVerification = _encryptionService.EncryptAES256(billingCreditPaymentInfo.CCVerification)
+                });
```

Comment:
Now you can use the new C# 11 [Raw String Literals](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/strings/#raw-string-literals):

```suggestion
            await connection.QueryFirstOrDefaultAsync("""
                UPDATE
                    BC.CCNumber = @CCNumber,
                    BC.CCExpMonth = @CCExpMonth,
                    BC.CCExpYear = @CCExpYear,
                    BC.CCVerification = @CCVerification
                FROM
                    [dbo].[BillingCredits] BC
                WHERE
                    IdBillingCredit = @billingCreditId
                """,
                new
                {
                    @billingCreditId = billingCreditId,
                    @CCNumber = _encryptionService.EncryptAES256(billingCreditPaymentInfo.CCNumber.Replace(" ", "")),
                    @CCExpMonth = billingCreditPaymentInfo.CCExpMonth,
                    @CCExpYear = billingCreditPaymentInfo.CCExpYear,
                    @CCVerification = _encryptionService.EncryptAES256(billingCreditPaymentInfo.CCVerification)
                });
```